src/apis/platformvm/api.ts

```diff
@@ -127,15 +127,31 @@ export class PlatformVMAPI extends JRPCAPI {
   /**
    * Fetches the AVAX AssetID and returns it in a Promise.
    *
+   * @param refresh This function caches the response. Refresh = true will bust the cache.
```

Comment:
Maybe add:

> Defaults to `true`

Reply:
This defaults to false, the "Refresh = true" comment explains the behavior.